spec/features/user_adds_a_new_landmark_spec.rb

```diff
@@ -6,23 +6,27 @@
 # # [] When adding a new landmark, if I fill out the form incorrectly and submit the form, I should see the form and be displayed the validation errors.
 
 feature "User visits new landmark path" do
+
   scenario "user adds a new landmark successfully" do
+    user_login
+    landmark = FactoryGirl.create(:landmark)
+
     visit new_landmark_path
 
     expect(page).to have_content "New Landmark Form"
 
-    fill_in 'Name', with: "Statue of Liberty"
-    fill_in 'Location', with: "New York"
-    fill_in 'Image', with: "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Statue_of_Liberty_7.jpg/500px-Statue_of_Liberty_7.jpg"
-    fill_in 'Description', with: "A colossal neoclassical sculpture on Liberty Island"
+    fill_in 'Name', with: landmark.name
+    fill_in 'Location', with: landmark.location
+    fill_in 'Image', with: landmark.image
+    fill_in 'Description', with: landmark.description
 
     click_button "Submit Landmark"
 
     expect(page).to have_content "Landmark added successfully!"
-    expect(page).to have_content "Statue of Liberty"
-    expect(page).to have_content "New York"
-    page.should have_selector ("img[src$='https://upload.wikimedia.org/wikipedia/commons/thumb/a/a1/Statue_of_Liberty_7.jpg/500px-Statue_of_Liberty_7.jpg']")
-    expect(page).to have_content "A colossal neoclassical sculpture on Liberty Island"
+    expect(page).to have_content landmark.name
+    expect(page).to have_content landmark.location
```

Comment:
@dreamingfurther Interesting, cause that conflicts with feedback I'd gotten previously where the reviewer preferred variables over hardcoded strings given how you might update your factories.